tests/strainMeasurement/plotVelocities.py

```diff
@@ -0,0 +1,656 @@
+#!/usr/bin/env/python
+
+import sys,traceback
+if sys.version_info[0] != 3:
+    print("This script is only python3 compatible!")
+    exit(1)
+
+
+import numpy as np
+import h5py
+from attrdict import AttrMap,AttrDict
+from argparse import ArgumentParser
+import os.path
+import json
+
+# Import Plot Settings
+sys.path.append( os.environ["PYTHONSCRIPT_DIR"] )
```

Comment:
The PlotSettings package and the PYTHONSCRIPT_DIR seem to be something specific to your setup.  At least I couldn't figure out what this is.